content/applications/websites/ecommerce/checkout_payment_shipping/ewallet_giftcard.rst

```diff
@@ -0,0 +1,122 @@
+======================
+eWallet and gift cards
+======================
+
+You can enable **eWallets** and **gift cards** for your customers on your eCommerce, which they can
+use to for their online and in-store shopping. To allow eWallets and gift cards for your eCommerce,
+head to :menuselection:`Website app --> eCommerce` and in the :guilabel:`Shop - Products` category,
+enable :guilabel:`Discounts, Loyalty & Gift Card`.
+
+eWallet
+=======
+
+eWallet allows customers to save their credits on their online account and use these credits as a
+payment method when buying items on your online store. As multiple redeems for the same order are
```

Comment:
Not sure to understand what you mean by "As multiple redeems for the same order are not possible in Odoo". 
And do we want to highlight what is not possible to do in Odoo? Should we just focus on what we can do?

Reply:
I meant multiple gift card redeems (added for clarity), and I think it should be documented as (IMO) it is a feature that could be expected by customers